marketplace/dev/setup_kubeconfig.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# If ~/mount/.kube/ exists ($KUBECONFIG is mounted) and ~/.kube does not
+# (we have not yet initialized), copy system kubectl configuration to
+# them to the default $KUBECONFIG location after adjusting system-specific
+# fields.
+if [[ -e "/root/mount/.kube/config" && ! -e "/root/.kube/config" ]]; then
+  mkdir -p /root/.kube
+
+  cat /root/mount/.kube/config \
```

Comment:
please add a comment clarifying what this is doing

Reply:
Done.